yarn.lock

```diff
@@ -295,7 +295,7 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
-"@firebase/app-types@0.6.1", "@firebase/app-types@0.x":
```

Comment:
Thanks for fixing this!!

Reply:
Haha this was by accident, didnt know i made this change